src/cache.py

```diff
@@ -0,0 +1,55 @@
+# |==========================================|
+# | Copyright (c) 2017-present Reece Dunham  |
+# | All rights reserved                      |
+# | Used with explicit permission            |
+# |==========================================|
+
+# Taken from a project that may become open source soon, please keep the copyright header
+
```

Comment:
```suggestion

from io import TextIOWrapper

```

`NameError: name 'TextIOWrapper' is not defined`

Reply:
Huh, should be in memory by default